modules/EsminiAdapter/src/esminiadapter.cpp

```diff
@@ -143,7 +143,6 @@ void EsminiAdapter::onStaticInitMessage(
 		RCLCPP_ERROR(me->get_logger(), e.what());
 		return;
 	}
-	me->InitializeEsmini();
```

Comment:
I would like to avoid moving this back here. Having it in the callback opens for race conditions when object control requests esmini data on init.